packages/shared/components/context-menu/ContextMenu.tsx

```diff
@@ -260,7 +279,10 @@ const ContextMenu = React.forwardRef((props: ContextMenuProps, ref) => {
 
       // fit
       if (top < document.body.scrollTop) {
-        top = document.body.scrollTop;
+        const marginTop = 16;
```

Comment:
Move from code to const and add description

Reply:
fFix, add  to const and add description